packages/manager/apps/dedicated/client/app/managedBaremetal/confirm-terminate/confirm-terminate.routes.js

```diff
@@ -13,6 +13,7 @@ export default /* @ngInject */ ($stateProvider) => {
     resolve: {
       goBack: /* @ngInject */ (goBackToDashboard) => goBackToDashboard,
       token: /* @ngInject */ ($transition$) => $transition$.params().token,
+      breadcrumb: () => null,
```

Comment:
Can we move the `breadcrumb` property to a top-level resolver so that it can just be overiden wherever required?

Reply:
I'm having trouble understanding, the breadcrumb is built according to the state tree, and adds a value for each state that compose this tree, if we define a top level property as I understand from your comment, we will only define it for the root state

Reply:
My bad... I am not thorough in the logic used for breadcrumb in `@ovh-ux/ng-ui-router-breadcrumb`. Thanks